src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccess.scala

```diff
@@ -48,4 +48,36 @@ trait DataAccess
       workflowFailureQuery.schema ++
       workflowMessageQuery.schema ++
       workflowAuditStatusQuery.schema
+
+  def truncateAll: WriteAction[Int] = {
+    // important to keep the right order for referential integrity !
```

Comment:
+1 what Biden said, though it'd be good to catch this list being:

a) incorrect; or
b) missing tables.

I think a) would just cause every test to barf horribly on cleanup. Not sure what would happen to b). Doublecheck both?